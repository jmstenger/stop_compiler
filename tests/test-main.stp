/*
 * test-main.stp
 * =============
 * Tests argc & argv functionality
 */

def main = (argc:Int, argv:String[]):Int {
	printf("%d\n", argc);
	printf("%s\n", argv[0]);
	return 0;
}
