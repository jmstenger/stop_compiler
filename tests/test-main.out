1
test-main.ll
